Python/Include/Generator.hpp

```diff
@@ -17,7 +18,9 @@ namespace Rover {
   */
   template<typename T>
   void export_generate(pybind11::module& module, const char* function_name) {
-    module.def(function_name, generate<T>);
+    module.def(function_name, [](T& generator) {
```

Comment:
What error happened here? Did you try ```&generate<T>```? What version of g++ are you using?

Reply:
G++ 8.2.0. I did; couldn't deduce template parameter Func at pybind11.h:812